nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/ProcessGroupResource.java

```diff
@@ -2013,7 +2014,13 @@ public Response createProcessGroup(
             // Step 2: Retrieve flow from Flow Registry
             final RegisteredFlowSnapshot flowSnapshot = getFlowFromRegistry(versionControlInfo);
 
-            // Step 3: Resolve Bundle info
+            // Step 3: Enrich version control info came from UI
+            final VersionedFlowCoordinates versionedFlowCoordinates = flowSnapshot.getFlowContents().getVersionedFlowCoordinates(); // If not null....
```

Comment:
@simonbence did you intend to check for nulls here? Looks like possibility of NPE?

Reply:
You are correct, I forgot the comment there from development. In the line below the null check for `versionedFlowCoordinates` happens. My understanding is that, `getFlowContents` should not return `null`. I will remove the comment if you are okay with this.